shipctl/x86_64/CentOS_7/shippable_jdk

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+service_action=$1
+JDK_VERSION=$2
+
+export_java_path() {
+  directory=$1;
+  if [ -d "$directory" ]; then
+    export JAVA_HOME="$directory";
+    export PATH="$PATH:$directory/bin";
+  else
+    echo "$JDK_VERSION is not supported on this image"
+    exit 99
+  fi
+}
+
+set_java_path() {
+  java_path=$1
+  if [ -f $java_path ]; then
+    sudo update-alternatives --set java $java_path
+  else
+    echo "$JDK_VERSION is not supported on this image"
+    exit 99
+  fi
+}
+
+set_javac_path() {
+  javac_path=$1
+  if [ -f $javac_path ]; then
+    sudo update-alternatives --set javac $javac_path
+  else
+    echo "$JDK_VERSION is not supported on this image"
+    exit 99
+  fi
+}
+
+shippable_jdk() {
+  if [ "$JDK_VERSION" == "openjdk7" ]; then
```

Comment:
we don't have java installed by default, so we should remove this. @shrivara  your opinion?